compute/cloud-client/src/test/java/compute/SnippetsIT.java

```diff
@@ -105,6 +108,10 @@ public static void setUp()
     compute.CreateInstance.createInstance(PROJECT_ID, ZONE, MACHINE_NAME_WAIT_FOR_OP);
     compute.CreateEncryptedInstance
         .createEncryptedInstance(PROJECT_ID, ZONE, MACHINE_NAME_ENCRYPTED, RAW_KEY);
+    compute.CreateInstanceWithRegionalLocation
```

Comment:
IIUC, the sample demonstrates how to create a regional instance template. Update the class name and the reference and parameters here.